framework/source/class/qx/event/handler/Mouse.js

```diff
@@ -375,6 +377,17 @@ qx.Class.define("qx.event.handler.Mouse",
 
       this.__fireEvent(domEvent, type, target);
 
+      /*
+       * In order to normalize middle button click events we
+       * need to fire an artificial click event if the client
+       * fires auxclick events for non primary buttons instead.
+       * 
+       * See https://github.com/qooxdoo/qooxdoo/issues/9268
+       */
+      if (type == "auxclick" && domEvent.button == 1) {
+        this.__fireEvent(domEvent, "click", target);
```

Comment:
Hmm. what is the feature you are asking for. Maybe I did not understand what you mean.